src/cpufreq.c

```diff
@@ -262,6 +280,8 @@ const struct feat_t *cpufreq_get_features(void)
         case T6021:
         case T6022:
             return t6020_features;
+        case T6031:
+            return t6031_features;
```

Comment:
same as T6021/T6022?

Reply:
For now, yes.  I assume it will end up slightly different in the end, which is why i didn't reuse the struct.